src/test/java/at/ac/tuwien/kr/alpha/solver/AbstractSolverTest.java

```diff
@@ -514,4 +512,97 @@ public void simpleNoPropagation() throws IOException {
 		Set<AnswerSet> answerSets = solver.collectSet();
 		assertEquals(expected, answerSets);
 	}
+
```

Comment:
Those tests are a good addition. Since they come with specific helper and construction methods and these are all pigeon hole specific, I suggest do move them to their own class.